src/post.pegjs

```diff
@@ -88,6 +90,22 @@ HTML_Comment
     type: "HTML_Comment",
     innerText: cs.join('')
   } }
+
+HTML_Tag_Void
+  = t:HTML_Tag_Open
+  & { return undefined !== {
+      'br': true,
+      'col': true,
+      'embed': true,
+      'hr': true,
+      'img': true,
+      'input': true
```

Comment:
they are specified yes but I didn't take the time to add them all. the reason I chose to ignore some of them is because I figured we don't want to allow `link` and `meta` tags inside of post content.

actually in HTML5 it's _forbidden_ to have a closing tag on these so I probably should update the grammar to reflect that